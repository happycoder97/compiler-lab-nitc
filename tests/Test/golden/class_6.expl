class A         
{
    int foo;
    int setFoo(int f) {
        write("SET");
        write(f);
        self.foo = f;
    }

    int printFoo() {
        write(self.foo);
    }
}

A a;

int main() {
    initialize();
    a = new(A);
    a.setFoo(123);
    a.printFoo();
    write("DONE");
}
