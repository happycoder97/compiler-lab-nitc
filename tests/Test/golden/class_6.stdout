SET
123
123
DONE
